bin/omarchy-menu-power

```diff
@@ -7,8 +7,7 @@ show_power_menu() {
 \u200D Relaunch
 \u2060󰜉 Restart
 \u2063󰐥 Shutdown"
-
-  local selection=$(echo -e "$menu_options" | wofi --show dmenu --width 150 --height 195 -O alphabetical --style ~/.config/wofi/select.css)
+  local selection=$(echo -e "$menu_options" | walker --dmenu --theme dmenu_150)
```

Comment:
Would be nice if going forward there was a way to pass width in on command line, so we don't have to make themes per menu size.